nemo/core/classes/modelPT.py

```diff
@@ -1025,7 +1025,7 @@ def on_validation_epoch_end(self) -> Optional[Dict[str, Dict[str, torch.Tensor]]
                 self.validation_step_outputs[dataloader_idx].clear()  # free memory
 
             if 'log' in output_dict:
-                self.log_dict(output_dict.pop('log'), on_epoch=True)
+                self.log_dict(output_dict.pop('log'), on_epoch=True, sync_dist=True, reduce_fx='mean')
```

Comment:
Just FYI, `reduce_fx` doesn't apply to `sync_dist`, it applies to `on_epoch`, ie reduction over step values ([docs](https://lightning.ai/docs/pytorch/stable/api/lightning.pytorch.core.LightningModule.html#lightning.pytorch.core.LightningModule.log). 
Since this change is in `on_validation_epoch_end()`, I'm not sure if the `reduce_fx='mean'` will do anything, but that's the default anyways.

Reply:
Also FYI, the reduction function for syncing over ranks is set [here](https://github.com/Lightning-AI/pytorch-lightning/blob/master/src/lightning/pytorch/core/module.py#L539)